plugins/woocommerce/includes/tracks/events/class-wc-products-tracking.php

```diff
@@ -546,4 +546,19 @@ public function possibly_add_tag_tracking_scripts( $hook ) {
 		}
 		WCAdminAssets::register_script( 'wp-admin-scripts', 'add-term-tracking', false );
 	}
+
+	/**
+	 * Check if the current process is importing products.
+	 *
+	 * @return bool True if importing, false otherwise.
+	 */
+	private function is_importing() {
+		// phpcs:disable WordPress.Security.NonceVerification.Missing
+		// Check if the current request is a product import.
+		if ( isset( $_POST['action'] ) && 'woocommerce_do_ajax_product_import' === $_POST['action'] ) {
+			return true;
```

Comment:
Do you know if this is the only way to check if an import is happening? Or does the importer also set a global variable?
I am ok with this, but just wanted to double check if there are other ways.
Initially I thought someone could also import using the CLI, but looks like that is not the case.